src/resources/nfts/index.ts

```diff
@@ -1,22 +1,32 @@
-import { QueryFunction, useQuery } from '@tanstack/react-query';
-import { QueryConfigWithSelect, createQueryKey } from '@/react-query';
-import { NFT } from '@/resources/nfts/types';
+import { UniqueAsset } from '@/entities';
+import { arcClient } from '@/graphql';
+import { NftCollectionSortCriterion, SortDirection } from '@/graphql/__generated__/arc';
+import { Network } from '@/helpers';
+import { QueryConfigWithSelect, createQueryKey, queryClient } from '@/react-query';
+import { AppState } from '@/redux/store';
 import { fetchSimpleHashNFTListing } from '@/resources/nfts/simplehash';
 import { simpleHashNFTToUniqueAsset } from '@/resources/nfts/simplehash/utils';
+import { QueryFunction, useQuery } from '@tanstack/react-query';
 import { useSelector } from 'react-redux';
-import { AppState } from '@/redux/store';
-import { Network } from '@/helpers';
-import { UniqueAsset } from '@/entities';
-import { arcClient } from '@/graphql';
 import { createSelector } from 'reselect';
-import { NftCollectionSortCriterion } from '@/graphql/__generated__/arc';
 
 const NFTS_STALE_TIME = 600000; // 10 minutes
 const NFTS_CACHE_TIME_EXTERNAL = 3600000; // 1 hour
 const NFTS_CACHE_TIME_INTERNAL = 604800000; // 1 week
 
-export const nftsQueryKey = ({ address, sortBy }: { address: string; sortBy: NftCollectionSortCriterion }) =>
-  createQueryKey('nfts', { address, sortBy }, { persisterVersion: 4 });
+export const nftsQueryKey = ({
+  address,
+  sortBy,
+  sortDirection,
+}: {
+  address: string;
+  sortBy: NftCollectionSortCriterion;
+  sortDirection: SortDirection;
+}) => createQueryKey('nfts', { address, sortBy, sortDirection }, { persisterVersion: 1 });
+
+export const invalidateAddressNftsQueries = (address: string) => {
+  queryClient.invalidateQueries(createQueryKey('nfts', { address }));
+};
```

Comment:
probably want to include sortBy and sortDirection here so that the query is properly invalidated